src/presentation/cli/CollectMetrics.ts

```diff
@@ -0,0 +1,47 @@
+import { promises as fs } from 'fs'
+import {
+  createPublicClient,
+  http
+} from 'viem'
+import { hemiSepolia, hemi } from 'hemi-viem'
+
+const chain = process.env['TESTNET'] ? hemiSepolia : hemi
+const filePath = process.env['METRICS_FILE_PATH'] ?? ''
+
+const client = createPublicClient({ chain, transport: http() })
+const blockExplorerUrl = chain.blockExplorers.default.url
+
+const execute = async (): Promise<void> => {
+  console.info('Hemi Connector | Collecting chain metrics...')
+
+  try {
+    const [blockNumber, stats] = await Promise.all([
+      client.getBlockNumber(),
+      fetch(
+        `${blockExplorerUrl}/api/v2/stats`)
+        .then(async response => await response.json())
```

Comment:
This `async/await` can be dropped as it is redundant. `.then()` only needs to receive a promise, and `response.json()` returns just that

```suggestion
      fetch(
        `${blockExplorerUrl}/api/v2/stats`)
        .then(response => response.json())
```

Reply:
![Captura de Tela 2025-01-02 às 14 40 59](https://github.com/user-attachments/assets/8aee0516-4fc0-4313-82be-55cb75df0e8b)

@gndelia TS eslint is suggesting to add `async/await`

Reply:
It seems that the rule is trying to prevent code from mixing the return of values with and without promises accidentally. That wouldn't be the case here as `response.json()` always returns a Promise, and there's no other code.

One option would be to disable the rule with an inline `eslint-disable-next-line` - the second option would be to add `async` (just so the rule passes), though `await` can be omitted.


```suggestion
      fetch(
        `${blockExplorerUrl}/api/v2/stats`)
        .then(async response => response.json())
```

Try any of those options, they both should work

Reply:
fixed